src/NServiceBus.AcceptanceTests/Routing/SubscriptionBehavior.cs

```diff
@@ -1,9 +1,9 @@
-﻿namespace NServiceBus.AcceptanceTests.PubSub
+﻿namespace NServiceBus.AcceptanceTests.Routing
```

Comment:
It is time to change the default namespace of acceptance tests to NServiceBus? All the usings are annoying